src/components/CartCard.jsx

```diff
@@ -39,25 +41,21 @@ export default function CartCard({ productData }) {
                             <button
                                 className="btn btn-icon button-round p-0 p-x-0_5"
                                 onClick={() =>
-                                    dispatchCart({
-                                        type: "DECREMENT_QUANTITY",
-                                        payload: productData,
-                                    })
+                                    decrementQuantity(productData, setIsLoading)
                                 }
+                                disabled={isLoading}
                             >
-                                {quantity===1?<TrashIcon/>:<DecrementIcon />}
+                                {qty === 1 ? <TrashIcon /> : <DecrementIcon />}
```

Comment:
You can use Prettier for formatting.

Reply:
I have used prettier extension maybe because of github preview its taking next line.